src/styles/_logo.scss

```diff
@@ -0,0 +1,9 @@
+.logo {
+  width: 115px;
+  height: 76px;
+  background-image: url(../images/logo.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  z-index: 10;
```

Comment:
does z-index work without position?

Reply:
yes, i dont know for sure how, but it does. i thought the position is static for default and "z-index" consider it 🙂
and when burger-menu is open, logo cover it